---
title: Workfront Fusion：シナリオ接続エラー
description: 特定のシナリオの実行が失敗し、「接続を検証できませんでした」というエラーメッセージが表示されます。
hidefromtoc: true
feature: Workfront Fusion
source-git-commit: a2d7e98c114ca65872e60c4d9b507f90d663e5df
workflow-type: tm+mt
source-wordcount: '46'
ht-degree: 8%

---

# Workfront Fusion：シナリオ接続エラー

特定のシナリオの実行が失敗し、「接続を検証できませんでした」というエラーメッセージが表示されます。

_最初に報告されたのは 2025年2月19日（PT）です。_
